apps/andi/lib/andi_web/live/ingestion_live_view/edit_ingestion_live_view.ex

```diff
@@ -193,9 +193,9 @@ defmodule AndiWeb.IngestionLiveView.EditIngestionLiveView do
     {:noreply, assign(socket, changeset: new_ingestion_changeset, unsaved_changes: true)}
   end
 
-  def handle_info({:update_dataset, id}, socket) do
+  def handle_info({:update_dataset, update}, socket) do
```

Comment:
does this update need to happen? Makes the variable much more vague as to what it could be?

Reply:
yes absolutely. We can change the name so it's more understandable.